### PR TITLE
14.0 mig delivery carrier label default

### PR DESCRIPTION
**Pull request by KKamaa**
migration to v14.0, see sample below; it contains default label and generating .csv docs all in one sample.

delivery-carrier

### CODE REVIEW

File: delivery_carrier_label_default/models/stock_picking.py
Comment (gfcapalbo) [inline delivery_carrier_label_default/models/stock_picking.py]: multi is deprecated in 14.0 remove or will gen error

**Review by gfcapalbo — CHANGES_REQUESTED** [review]
Multi Decorator is deprecated in v14, please remove, 
the rest looks consistent with the last known version on v11.
I see no differences in the report.render method, so it should work the same.

Approved after decorator removal.

File: delivery_carrier_label_default/__manifest__.py
Comment (gfcapalbo) [inline delivery_carrier_label_default/__manifest__.py]: redundant, it is already installed via the dependency chain below:

base_delivery_carrier_label -> delivery_carrier_info -> delivery -> sale_stock ->stock_account -> [ stock ]

**Review by gfcapalbo — CHANGES_REQUESTED** [review]

**Comment by KKamaa** [comment]
@OCA/community-maintainers can someone do a check on this please

**Comment by KKamaa** [comment]
@thomaspaulb we are currently using this in our own 14.0 migration its labelled as stale

**Comment by sebalix** [comment]
@KKamaa Hello, we didn't saw your PR this year (the title was skipping the `_` in the name of the module so we didn't find it), so we did this one earlier this year: https://github.com/OCA/delivery-carrier/pull/456

I compared both migration commits and there are quite some changes between them, I can't say which one is better. What I know is we are using the linked PR above in production for some months now without issues, can you have a look at it please?

**Comment by sebalix** [comment]
@sonhd91 Can you have a look on this one too?

**Pull request closed without merging**
